### ISSUE TITLE
Two out of three APIs don't work

### ISSUE THREAD

**Issue by Discostu36** [zlant/parking-lanes#115]
This tool only works with overpass turbo. The other two APIs give errors:

overpass-vk: CORS Cross-Origin Error
OSM API: HTTP Error 400 Bad Request

**Comment by zlant** [comment]
Please check browser extensions

**Comment by Discostu36** [comment]
You are right, deactivating uBlock origin and Firefox Enhanced Tracking Protection solves the issue.

**Comment by Discostu36** [comment]
Maybe code can be changed in a way that this also works without having to disable tracking protection etc.

**Comment by kayD** [comment]
For me (as of 2023-06-20), the default (overpass-vk) does never work ("Unknown error, please try again"). The two others work for me.
